internal/connectors/db/yql/queries/write.go

```diff
@@ -335,16 +335,11 @@ func BuildUpdateBackupScheduleQuery(schedule types.BackupSchedule, index int) Wr
 	d.AddUpdateId(table_types.StringValueFromString(schedule.ID))
 	d.AddValueParam("$active", table_types.BoolValue(schedule.Active))
 	d.AddValueParam("$crontab", table_types.StringValueFromString(schedule.ScheduleSettings.SchedulePattern.Crontab))
-
-	if len(schedule.SourcePaths) > 0 {
-		d.AddValueParam("$paths", table_types.StringValueFromString(strings.Join(schedule.SourcePaths, ",")))
-	}
-	if len(schedule.SourcePathsToExclude) > 0 {
-		d.AddValueParam(
-			"$paths_to_exclude",
-			table_types.StringValueFromString(strings.Join(schedule.SourcePathsToExclude, ",")),
-		)
-	}
+	d.AddValueParam("$paths", table_types.StringValueFromString(strings.Join(schedule.SourcePaths, ",")))
```

Comment:
We need to somehow verify that for each `WithUpdateBackupSchedule` call, correct `SourcePaths` and  `SourcePathsToExclude` are set. It is hard, maybe we need some kind of integration test for schedules, that verifies that non-empty `SourcePaths` are not overrided by empty strings

Reply:
I looked through the code and it seems that it is ok for now, but we need a ticket for adding schedules to integration test